pkg/kv/iterators.go

```diff
@@ -0,0 +1,275 @@
+package kv
```

Comment:
Refactoring code by moving it between files in a PR with other changes, makes it almost impossible to understand if any previous code was changed.

Reply:
Code was not moved - file name was refactored. We have this issue with every KV package we implement unfortunately